docker/common.yml

```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=True
+   - SYNDICATION=True
+   - MONGO_DBNAME=liveblog
+   - SUBSCRIPTION_LEVEL=network
+   - LIVEBLOG_CACHE_TYPE=simple
+   - LIVEBLOG_CACHE_REDIS_URL=redis://redis:6379/12
```

Comment:
One last comment. Now that `LIVEBLOG_CACHE_TYPE=simple` we don't need the `LIVEBLOG_CACHE_REDIS_URL ` so please just remove it. Thanks

Reply:
ok, will remove LIVEBLOG_CACHE_REDIS_URL